searchlib/src/vespa/searchlib/features/attributefeature.cpp

```diff
@@ -413,10 +413,20 @@ createTensorAttributeExecutor(const IAttributeVector *attribute, const vespalib:
 
 }
 
-search::fef::FeatureExecutor &
-AttributeBlueprint::createExecutor(const search::fef::IQueryEnvironment &env, vespalib::Stash &stash) const
+void
+AttributeBlueprint::prepareSharedState(const fef::IQueryEnvironment & env, fef::IObjectStore &) const
+{
+    //Ensure that we remember it the whole query
+    const IAttributeVector * attribute = env.getAttributeContext().getAttribute(_attrName);
+    if (_attribute == nullptr) {
+        _attribute = attribute;
```

Comment:
The query environment attribute context will make sure that this pointer is valid during query execution. This pointer will be used for multiple queries, which seems very unsafe and undefined. Please put this pointer in the object store, which stores per-query state.